### PR TITLE
Modify bin/main.rb

### PR DESCRIPTION
**Pull request by jucora**

### CODE REVIEW

File: bin/main.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
+module UserInteraction
+    def get_rol
+        puts "\nWELCOME TO THE TIC TAC TOE GAME"
+        puts "==========================="
+        puts "Which player do you prefer?"
+        puts "1 for X or 2 for O"
+        puts "==========================="
+        $rol = gets.chomp.to_i
```
Comment (maelfosso) [inline bin/main.rb]: Sorry but the **global variables** is not allowed

File: bin/main.rb [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    def input_cell
+        @cell = nil
+        puts "\nSelect a number in the cell"
+        @cell = gets.chomp.to_i
+    end
+    def again?
+        puts "##################"
+        puts "#PLAY AGAIN? (y/n)"
+        puts "##################"
+        $again = gets.chomp
```
Comment (maelfosso) [inline bin/main.rb]: No **global variables** please

File: bin/main.rb [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+        if $again == "y" || $again == "Y"
+            player = Player.new
```
Comment (maelfosso) [inline bin/main.rb]: Where does the `Player` class come from please ?

**Review by maelfosso — CHANGES_REQUESTED** [review]
Hello @jucora and @Ashah15 

# Nice work :clap: 

It's really surprising to see how you succeed in producing that code at first step.
Awesome job :+1: 

Even though your code is great, there is still some changes to do
- **REQUIRED** No global variables
- You are not supposed to have classes here, like `Player`
- According to the description of the Milestone, you supposed to `Display prompts for users, collect user input and only mock the logic (implementing the logic will be the next step).` but in your code there is no display, no input collected.

At this step, imagine that the game is working. You're welcoming the user with a message, you ask him for their names, displaying the board and ask him the cell where he wants to play and simulate (even randomly) the state of the game, the winner and etc... **You noticed that their is no logic**.
Also, don't think about class here ( **OPTIONAL** )

Now (after that imagination), only in the `bin/main.rb`, implement all the interactions you imagine and the game state simulation.
It's briefly what you have to do.

Kindly, do the changes and submit a new code review request.
Thanks

**Comment by Ashah15** [comment]
the player class and the global variables were removed as per the review received. We added some interactivity but the rest will be in the next milestone.

File: bin/main.rb (hunk 1 above)
Comment (jucora) [reply bin/main.rb]: done

File: bin/main.rb (hunk 2 above)
Comment (jucora) [reply bin/main.rb]: done

File: bin/main.rb (hunk 3 above)
Comment (jucora) [reply bin/main.rb]: done

**Review by bellom — APPROVED** [review]
Hi Team,
**Great work making all the necessary corrections from the previous review.**
Kindly be aware that it is **`Required`** of you to add linter in every project and you should be able to run the check on your `Github` PR page, example below;
image
I will approve this PR because i trust you going to add the `linter` in the next milestone.
## Congratulation :+1:  
**_Go ahead and merge to dev_** :+1: 
Goodluck in your next milestone :slightly_smiling_face:

**Pull request merged**